openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaAuthConfigGenerator.java

```diff
@@ -646,6 +694,29 @@ public AssignmentStatementNode getApiKeyAssignmentNode() {
         return recordFieldNodes;
     }
 
+    /**
+     * Generate fields for`AuthConfig` record when combination of ApiKey and HTTPOrOAuth authentication is supported.
+     *
+     * <pre>
+     *     # Auth Configuration
+     *     http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig|ApiKeysConfig auth;
+     * </pre>
+     * @return {@link List<Node>}   fields of AuthConfig record
+     */
+    private List<Node> getAuthConfigRecordFields() {
+        List<Node> recordFieldNodes = new ArrayList<>();
+        Token semicolonToken = createToken(SEMICOLON_TOKEN);
+
+        MetadataNode authMetadataNode = getMetadataNode("Auth Configuration");
+        IdentifierToken authFieldName = AbstractNodeFactory.createIdentifierToken(escapeIdentifier(AUTH));
+        TypeDescriptorNode authFieldTypeNode = createSimpleNameReferenceNode(
+                createIdentifierToken(getAuthFieldTypeName() + "|ApiKeysConfig"));
```

Comment:
```suggestion
                createIdentifierToken(getAuthFieldTypeName() + "|" + GeneratorConstants.API_KEYS_CONFIG"));
```
Or we can directly use `createUnionTypeDescriptorNode` as below
```suggestion
                createUnionTypeDescriptorNode(
                createSimpleNameReferenceNode(createIdentifierToken(getAuthFieldTypeName())),
                createToken(PIPE_TOKEN), createSimpleNameReferenceNode(createIdentifierToken(API_KEYS_CONFIG)));
```

Reply:
Done